### PR TITLE
Added default import values and cleanup

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,14 +116,19 @@ To connect to a cloud instance of CloudVision you must set the following variabl
 
 When syncing from CloudVision, this plugin will create new devices that do not exist in Nautobot. In order for this to work properly, you must provide the following default value sin the nautobot config file.
```
Comment (chipn) [inline README.md:117]: Typo here unrelated to your PR: "value sin"

File: README.md [hunk 2]
```diff
@@ -104,14 +116,19 @@ To connect to a cloud instance of CloudVision you must set the following variabl
...
+- `from_cloudvision_default_device_role_color` string: The default color to assign to the default role. Defaults to `ff0000`.
+- `from_cloudvision_default_device_status`: string: The default status used when the syncing creates new devices in Nautobot. Defaults to `cloudvision_imported`.
+- `from_cloudvision_default_device_status_color`: string: The default color to assign to the default status. Defaults to `ff0000`.
+
+> When these variables are not defined in the plugin settings, the plugin will use the default values mentioned.
 
 Lastly, when a device exists in Nautobot but not in CloudVision, this plugin can either delete or leave the device in Nautobot. That behavior can be set with the following variable in the nautobot config file.
 
-- `delete_devices_on_sync_cv_source` boolean (default False): If true, this will delete devices in Nautbot that do not exist in CloudVision when syncing from CloudVision.
+- `delete_devices_on_sync_cv_source` boolean (default False): If true, this will delete devices in Nautbot that do not exist in CloudVision when syncing from CloudVision. Defaults to `False`.
```
Comment (chipn) [inline README.md:129]: Should we have a note that this only applies to Arista devices?

File: README.md (hunk 1 above)
Comment (qduk) [reply README.md]: Fixed. Thanks

File: README.md (hunk 2 above)
Comment (qduk) [reply README.md]: Yeah, that's a good idea. I put it in the variable description but I also thought about adding a `>` line under it. Thoughts?

**Review by chipn — APPROVED** [review]
LGTM

**Pull request merged**